README.md

```diff
@@ -60,12 +60,33 @@ Project has no opportunity to authorize new users as edX's instance student.
 So you need to do this manually:
 1. Go to admin panel, then to `Edx telegram users` model (table).
 2. Make relationship between Telegram user and edX user in field `student`.
-3. Add telegram `id` and telegram `nickname`.
+3. Add telegram `id` and telegram `username` (optional move, it\'s can be empty - `''`).
 
-If you don`t know telegram id and nickname, just go ahead and log this, i.e:
+If you don't know telegram id and nickname, just go ahead and log this there
+https://github.com/raccoongang/edx_telegram_bot/blob/master/edx_telegram_bot/edx_telegram_bot.py#L169:
 ```
 def help_command(self, bot, update):
     print update.message
 ```
 
-`update.message` will show you all keys you need like `id`, `nick`, `chat id`, `message id`, etc.
+`update.message` example of print:
+```
+Update {'message': {'migrate_to_chat_id': 0, 'delete_chat_photo': False, 'new_chat_photo': [], 
```

Comment:
@z4y4ts, why i need to use `pprint.pprint`, that needed additional `import pprint`?
Dev just gonna get data one time.